.golangci.yml

```diff
@@ -20,6 +20,12 @@ issues:
     - text: "ST1003:"
       linters:
         - stylecheck
+    # spec.RewriteConstants replaced by Variables API:
+    # https://github.com/cilium/ebpf/pull/1564
+    # The change should be in next release cilium/ebpf v0.17.0
+    - text: "SA1019: .*RewriteConstants is deprecated"
+      linters:
+        - staticcheck
```

Comment:
I would prefer the inline version

Reply:
I've put it here because there are 10 occurrences in different go files. But I can try to move it to the go files if you think it is better?

Reply:
> I've put it here because there are 10 occurrences in different go files. But I can try to move it to the go files if you think it is better?

Hmm, that could change my mind